LimeBean.Tests/BeanTests.cs

```diff
@@ -114,6 +114,17 @@ public class BeanTests {
         }
 
         [Fact]
+        public void Columns() {
+            var bean = new Bean();
+            bean["a"] = 1;
+            bean["b"] = 2;
+            bean["c"] = null;
+
+            var expect = new[] { "a", "b", "c" };
```

Comment:
I'll just be verbose instead of trying to be clever, which is what brought on the above :sweat_smile: